internal/cmdapply/cmdapply.go

```diff
@@ -199,10 +199,19 @@ func runApply(r *Runner, invInfo inventory.InventoryInfo, objs []*unstructured.U
 
 	// Run the applier. It will return a channel where we can receive updates
 	// to keep track of progress and any issues.
-	if err := r.Applier.Initialize(); err != nil {
+	poller, err := factory.NewStatusPoller(r.factory)
```

Comment:
Not much familiar with the code, but `factory.NewSomething(r.factory)` feels a bit odd to read as code reader. May be `status.NewPoller(r.factory)` or `poller.NewStatusPoller(r.factory)` ?

Reply:
Good point. I have used an alias for the import so it is now `status.NewStatusPoller(r.factory)`